disnake/embeds.py

```diff
@@ -238,73 +234,64 @@ def from_dict(cls: Type[E], data: Mapping[str, Any]) -> E:
             The dictionary to convert into an embed.
         """
         # we are bypassing __init__ here since it doesn't apply here
-        self: E = cls.__new__(cls)
+        self = cls.__new__(cls)
 
         # fill in the basic fields
 
-        self.title = data.get("title", EmptyEmbed)
-        self.type = data.get("type", EmptyEmbed)
-        self.description = data.get("description", EmptyEmbed)
-        self.url = data.get("url", EmptyEmbed)
+        self.title = data.get("title", None)
+        self.type = data.get("type", "rich")
```

Comment:
why is type always set to rich?

Reply:
Initially that was added to avoid having an optional `type`, but given that it was `EmptyEmbed` before, optional does make sense here.
b613d32c5e87ff534d180e212d280e9f48f8d081